### PR TITLE
README.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by darosior**
Starting the documentation effort with those two main files. I'll address both a dedicated document about building the software and another dedicated document about recovering from descriptor backup in a follow-up.

### CODE REVIEW

**Comment by darosior** [comment]
I should discuss signing devices actually...

File: README.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# Liana
+
+*The missing safety net for your bitcoins*.
```
Comment (edouardparis) [inline README.md:3]: *a hold to save you if you fall*

**Review by edouardparis — APPROVED** [review]
Very good for a first write !

**Comment by darosior** [comment]
Thanks for proofreading. I've added a line about how you need a signing device to use Liana, and only Ledger / Specter are supported. I'm going to merge this.

File: README.md (hunk 1 above)
Comment (darosior) [reply README.md:3]: Oh sorry i missed it, didn't mean to merge without answering it. :sweat_smile: 

I prefer my catchprase though. We can bring it up with the others.

**Pull request merged**